Travis CI file & user table merge

lindyrp opened the pull request:
I finally have Travis CI passing on my forked repo! So time to get this party started on the master. Summary of commits:
* update .travis.yml file to resolve test database errors
* when trying to resolve the above, i discovered there were two tables named "user" when a db:setup command that I ran merged them into one, and dropped two of the fields;
* so I ran a new migration adding those two fields back into the new, unified user table.

Looking for feedback or where I might have done this better, since figuring out the database error messages I was getting from Travis CI was a real head-scratcher.

AlekstheRealist commented on db/schema.rb:
```diff
@@ -36,14 +36,6 @@
   create_table "users", force: true do |t|
-    t.integer  "user_id"
-    t.string   "email"
-    t.string   "name"
```
Are we removing "name" from out database altogether?

ruishine replied:
No, because it was added in another migration created above. I'm not too sure about this.

AlekstheRealist commented on db/schema.rb:
```diff
@@ -48,6 +48,8 @@
     t.datetime "created_at",                          null: false
     t.datetime "updated_at",                          null: false
+    t.integer  "user_id"
+    t.string   "name"
```
Ah I see you added back "name", nevermind then

AlekstheRealist commented:
👍 Looks good

ruishine commented on db/migrate/20160618224927_merge_user_tables.rb:
```diff
@@ -0,0 +1,6 @@
+class MergeUserTables < ActiveRecord::Migration
```
I'm a little confused by this migration because the User table already has user_id as integer as well as name as string from when I created and merged the database structure =/

ruishine commented:
I'm having trouble with the changes in database.  Are we replacing the existing columns because they are broken for travis CI?  I see we are replacing with same columns though...

lindyrp commented:
Yeah, I really struggled with this one and was hoping to get Ilya's thoughts for sure. The issue with the databases is that we had two *separate* tables called "user," which I didn't discover in the schema.rb until after I fixed the database error that Travis CI was throwing me, and found that the fix had merged them and dropped two of the columns (name and user_id). So I backtracked and instead of recreating the entire table, added the two dropped fields into the already-existed "user" table that was created when Aleks added Devise.

lindyrp commented:
Now that the database issue has been resolved, closing this PR and will submit a separate one for .travis.yml file changes.
